src/Symfony/Component/Messenger/composer.json

```diff
@@ -22,6 +22,7 @@
     "require-dev": {
         "psr/cache": "^1.0|^2.0|^3.0",
         "symfony/console": "^5.4|^6.0",
+        "symfony/clock": "^6.2",
```

Comment:
Should be before console

Reply:
Oh, by bad.
Hovewer it doesn't help with tests: "Error: Class "Symfony\Component\Clock\MockClock" not found"